src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common/matmul.cpp

```diff
@@ -271,7 +100,11 @@ std::vector<CPUSpecificParams> filterSpecificParams_Brgconv1x1() {
 std::vector<CPUSpecificParams> filterSpecificParams_MLAS() {
     // replace with mlas primitive type
     std::vector<CPUSpecificParams> specificParams;
+#if defined(OPENVINO_ARCH_ARM64) || defined(OPENVINO_ARCH_ARM)
```

Comment:
I think we need to have a separate instance under x86/mlas to avoid #ifdef

Reply:
fixed